src/components/storage/Common.jsx

```diff
@@ -143,7 +144,11 @@ export const getNewPartitioning = async ({
 
     const part = await createPartitioning({ method });
 
-    if (currentPartitioning?.method === method && method === "AUTOMATIC" && currentPartitioning.requests[0].encrypted) {
+    if (storageScenarioId === "home-reuse") {
+        await partitioningSetHomeReuse({ partitioning: part });
+    }
+
+    if (currentPartitioning?.method === method && method === "AUTOMATIC" && storageScenarioId !== "home-reuse" && currentPartitioning.requests[0].encrypted) {
```

Comment:
Can you do a multiliner with this please? 
It's too long now.

Reply:
Updated